source/process/marketing-guidelines.rst

```diff
@@ -0,0 +1,68 @@
+
+In order to achieve this overriding goal, we must:
+
+ 1. Increase awareness of Mattermost as a solution to key challenges facing organizations as measured by new visits to mattermost.com
+ 2. Reduce the time for organizations to decide on deploying open source and commercial versions through educational content on product value 
+ 3. Increase the number of organizations deploying Mattermost swiftly, efficiently and effectively 
+ 4. Increase the compellingness and trustworthiness of Mattermost product and marketing content as measured by increase in return visits 
+
+The biggest uplift to inbound marketing will come from being featured in online publications such as The Hacker News.
+
+Principles
+---------------------------------------------------------
+
+In order to achieve these goals, our writing must be:
+
+ 1. Magnetic to IT. We need headlines and openings that draw in IT audiences (these include Linux admins, security experts, directors of end-user computing, CIOs). Bear in mind that, although they work in IT, these audiences are typically more interested in the business benefits of technology than technical features. 
+ 2. Appealing to secondary audiences. These include open source and in-house developers and end users at highly regulated companies.
+ 3. Part of our brand. Our style and message must be consistent over the long term. Therefore, all of our content, irrespective of purpose or subject-matter, must pull the reader in the same direction and reinforce our key messages. 
+ 4. High quality. Quality in writing reflects the professionalism of our organization. We pay attention to the rules of grammar and spelling.
+ 5. Clear & transparent. We aim to communicate simply, using language that is familiar to our audiences. People don't buy what they don't understand!
+ 6. International. Mattermost has customers around the world and in many cases English is their second language. Use standard American English but do not use metaphors or expressions that people outside of North America are unlikely to understand.
+ 7. Avoiding unintentionally controversy. We might want to stir up a bit of controversy now and again – but only on relevant technological and business topics. Avoid subjects or analogies that might provoke an emotional or hostile response: religion, politics, warfare, personal relationships etc. 
+ 8. Fit for purpose. Is your content designed to persuade (e.g. an opinion piece), promote (e.g. a product description) or instruct (e.g. to guide a user through procedural steps)? Adjust your tone and your language accordingly.
+
+
+Clear? So now let's look at our 12 tips for brilliant writing. To win the reader's attention, and hold it, our content must be a pleasure to read. Our words must sparkle. Yet at the same time, it should all seem effortless; it can be a turnoff if it appears that the reader is trying too hard. 
+
+These guidelines help you to achieve the right balance. 
```

Comment:
"help you achieve", also the tense seems weird. Maybe "These guidelines will help you achieve", or "these guidelines are in place to help you achieve"